running-locally.adoc

```diff
@@ -1,64 +1,46 @@
 [[running-the-demo-raincatcher-solution-locally]]
-Running The Demo Raincatcher Solution Locally
----------------------------------------------
-
-[[overview]]
-Overview
-~~~~~~~~
-
-This guide presents the steps necessary to:
-
-* Run the Raincatcher demo solution locally.
-* Edit Raincatcher modules and have the results reflected in the demo
-solution.
+= Running The Demo Raincatcher Solution Locally
 
 [[requirements]]
-Requirements
-~~~~~~~~~~~~
+== Requirements
 
 [[rhmap]]
-RHMAP
-~~~~~
+=== RHMAP
 
-The demo solution currently requires an instance of RHMAP and will not work properly without it. It can be a local or remote instance of RHMAP, but must be configured as outlined in the Raincatcher link:getting-started.adoc[Getting Started Guide]. This allows the local *raincatcher-demo-cloud* app to access
+The demo solution currently requires an instance of RHMAP. It can be a local or remote instance of RHMAP, but must be configured as outlined in the Raincatcher link:getting-started.adoc[Getting Started Guide] so that the local *raincatcher-demo-cloud* app can access the following RHMAP features:
 
-- Forms
-- Themes
-- Drag & Drop App Config
+* Forms
+* Themes
+* Drag & Drop App Configuration
 
 
+// TODO are there any version requirements?
 [[mongodb]]
-MongoDB
-^^^^^^^
+=== MongoDB
 
-The *raincatcher-demo-cloud* app and *raincatcher-demo-auth* app requires a running MongoDB instance. See the link:https://docs.mongodb.com/manual/installation/[MongoDB Installation Guide] for details on how to install MongoDB.
+The *raincatcher-demo-cloud* app and *raincatcher-demo-auth* app requires a local running MongoDB instance. See the link:https://docs.mongodb.com/manual/installation/[MongoDB Installation Guide] for details on how to install MongoDB.
 
 [[redis]]
-Redis
-^^^^^
+=== Redis
 
-The *raincatcher-demo-cloud* app requires a running instance of Redis. See the link:https://redis.io/topics/quickstart[Redis Quick Start Guide] for details on how to insall Redis manually, or use your system's package manager.
+The *raincatcher-demo-cloud* app requires a local running instance of Redis. See the link:https://redis.io/topics/quickstart[Redis Quick Start Guide] for details on how to install Redis, or use your system's package manager.
 
+// TODO is this optional
```

Comment:
The `raincatcher-cli` tool is recommended, but indeed optional. You can clone the repositories and start the apps with whatever method you prefer.